src/gui/sharemanager.cpp

```diff
@@ -583,7 +604,8 @@ QSharedPointer<LinkShare> ShareManager::parseLinkShare(const QJsonObject &data)
         url,
         expireDate,
         note,
-        data.value("label").toString()));
+        data.value("label").toString(),
+        static_cast<bool>(data.value("hide_download").toInt())));
```

Comment:
Why not just use `toBool`?

Reply:
Fair enough, though maybe a better way of writing this would be

```suggestion
        data.value("hide_download").toInt() == 1);
```